BrainPortal/app/views/tool_configs/_form_fields.html.erb

```diff
@@ -67,6 +67,14 @@
         </div>
       <% end %>
 
+      <% t.edit_cell(:inputs_readonly, :header => "Read only input files", :show_width => 2) do |f| %>
```

Comment:
Within the web page, let's not use the 'inputs_readonly' phrasing, it's super ugly. Let's show this as 'Does not modify its inputs files'